auth-mailchimp-sync/extension.yaml

```diff
@@ -51,6 +53,7 @@ resources:
       eventTrigger:
         eventType: providers/firebase.auth/eventTypes/user.create
         resource: projects/${PROJECT_ID}
+
```

Comment:
I'd prefer to not add line breaks between items in a list (`-` indicates items in a list). That's very non-standard for yaml files, although it does work (much to my surprise).

Reply:
The other extension.yaml files with multiple resource objects have line breaks.  Also, when there are multiple parameters, we also have line breaks.

In most of the yaml files used internally, we use line breaks, as well.

Reply:
Oh interesting, I never noticed that. Ya learn something new everyday!

Reply:
ya, yaml files are flaky, but with this, there're not bad :-)